datajunction-ui/src/app/services/DJService.js

```diff
@@ -1,12 +1,24 @@
 import { MarkerType } from 'reactflow';
+import Cookies from 'js-cookie';
 
+const getAuthorizationHeaders = () => {
+  const bearer_token = Cookies.get('__dj');
+  return new Headers({
+    Authorization: `Bearer ${bearer_token}`,
+    'Content-Type': 'application/x-www-form-urlencoded',
+  });
+};
 const DJ_URL = process.env.REACT_APP_DJ_URL
   ? process.env.REACT_APP_DJ_URL
   : 'http://localhost:8000';
 
 export const DataJunctionAPI = {
   node: async function (name) {
-    const data = await (await fetch(DJ_URL + '/nodes/' + name + '/')).json();
+    const data = await (
+      await fetch(DJ_URL + '/nodes/' + name + '/', {
```

Comment:
Nit, it's cleaner to do:
```suggestion
      await fetch(`${DJ_URL}/nodes/${name}/`, {
```

Same below in the other `fetch`es.